spec/lib/unwrappr/researchers/github_comparison_spec.rb

```diff
@@ -37,43 +37,74 @@ module Researchers
         context 'given a Github repo' do
           let(:github_repo) { 'envato/unwrappr' }
 
-          context 'given the repo has "vx.x.x" tags' do
-            before do
-              allow(client).to receive(:compare)
-                .with('envato/unwrappr', 'v1.0.0', 'v1.1.0')
-                .and_return(response)
-              allow(client).to receive(:compare)
-                .with('envato/unwrappr', '1.0.0', '1.1.0')
-                .and_return(nil)
-            end
+          context 'given the gem is added' do
+            let(:base_version) { nil }
+            let(:head_version) { GemVersion.new('1.1.0') }
 
-            it 'returns the data from Github' do
-              expect(research).to include(github_comparison: response)
+            it "doesn't add data from Github" do
```

Comment:
The point of this PR is to ensure we don't call GitHub right ? not just not including the data. 
GitHub is an external part of this gem (obviously) I think you could add an extra `its` in here.. something like..
```ruby
it "doesn't contact GitHub a result we already know" do
  expect(client).to_not have_received(:compare)
end
```
or similar.. what do you think?

Reply:
Great idea.

Reply:
Added in 72fd8bb